infra-as-code/bicep/azure-firewall.bicep

```diff
@@ -120,7 +120,8 @@ resource azureFirewallPolicy 'Microsoft.Network/firewallPolicies@2024-05-01' = {
     }
```

Comment:
Please apply a Production readiness change bit in the network rules above.

Reply:
here it is @ckittel 

done | addressed from a0e20dfa82fdd921101a79d704c6e87e06e8ac23

Reply:
I only see it in the network rule